src/features/multichain/helpers/utils.ts

```diff
@@ -1,5 +1,8 @@
 import type { DecodedDataResponse } from '@safe-global/safe-gateway-typescript-sdk'
 
 export const isChangingSignerSetup = (decodedData: DecodedDataResponse | undefined) => {
-  return decodedData?.method === 'addOwnerWithThreshold' || decodedData?.method === 'removeOwner'
+  return (
+    decodedData?.method &&
+    ['swapOwner', 'addOwnerWithThreshold', 'removeOwner', 'changeThreshold'].includes(decodedData?.method)
```

Comment:
it would be nice if could we move the declaration of this array to be outside the function body, just to avoid having this array re-created on every function call.

Reply:
This hook isn't needed anymore after the other requested changes